openssh/files/sshd_config

```diff
@@ -77,7 +89,7 @@
 # What ports, IPs and protocols we listen for
 {{ option('Port', 22) }}
 # Use these options to restrict which interfaces/protocols sshd will bind to
-{{ option('ListenAddress', ['::', '0.0.0.0']) }}
+{{ option('ListenAddress', ['::', '1.0.0.0']) }}
```

Comment:
Aiee! That can't be right!

Sorry, @alxwr, I missed that. I'm not really in a position to change it any more, but thought I'd give you a heads up.

Reply:
https://github.com/saltstack-formulas/openssh-formula/pull/96#issuecomment-453072904